README.md

```diff
@@ -152,28 +162,41 @@ with the goal of eventual inclusion into Nix or Nixpkgs.*
 ![RFC Process](./rfcs/0036-rfc-process.png)
 ![Review Process](./rfcs/0036-review-process.png)
 
+### Tips
+
+#### Create Prototypes
+In case your RFC is a technical proposal, you might want to prepare a
+prototype of your idea to firstly make yourself aware of potential pitfalls
+and also help reviewers understand the RFC. Code may be able to explain some
+issues in short.
+
+###### Motivation First
+In any case, it can be a good idea to elaborate the motivation first, while
+leaving the PR in draft. This helps to ensure, that when the solution is
+discussed in the future, interested participants can have a clear and refined
+picture of the problem that you try to address.
 
 ## The RFC life-cycle
 
-Once an RFC is accepted the authors may implement it and submit the feature as a
-pull request to the Nix or Nixpkgs repo. Being accepted is not a rubber stamp,
-and in particular still does not mean the feature will ultimately be merged; it
-does mean that in principle all the major stakeholders have agreed to the
-feature and are amenable to merging it. In general though this means that the
-implementation will be merged as long as there are no substantial technical
+Once an RFC is accepted the authors may implement it and for example submit the
+feature as a pull request to the Nix or Nixpkgs repo. Being accepted is not a
+rubber stamp, and in particular still does not mean the feature will ultimately
+be merged; it does mean that in principle all the major stakeholders have agreed
+to the feature and are amenable to merging it. In general though this means that
+the implementation will be merged as long as there are no substantial technical
 objections to the implementation.
```

Comment:
Maybe `s/feature/change/`? It may also make sense to indicate that not all RFCs prompt change. How about:

> Once an RFC is accepted it may be implemented if any further work is required. This may be via a pull request to the nix or Nixpkgs repo or via other means. Being accepted is not a rubber stamp, and in particular still does not mean the change will ultimately be enacted; it does mean that in principle all the major stakeholders have agreed to the
change and are amenable to its implementation as long as there are no substantial technical objections to the implementation.